server/models/device.js

```diff
@@ -0,0 +1,22 @@
+import mongoose from 'mongoose';
+
+const DeviceSchema = new mongoose.Schema({
+  name: {
+    type: String,
+    required: true
+  },
+  type: {
+    type: String,
+    required: true,
+    enum: ['email', 'phone', 'sms', 'user']
+  },
+  address: {
```

Comment:
I'm assuming address means email address, phone number, etc. If that's the case, I don't think address is the best descriptor.